pkgs/development/python-modules/tensorboard/default.nix

```diff
@@ -26,7 +26,7 @@ buildPythonPackage rec {
   pname = "tensorboard";
   version = "2.11.0";
   format = "wheel";
-  disabled = pythonOlder "3.6" || pythonAtLeast "3.11";
+  disabled = pythonOlder "3.6";
```

Comment:
1. IMO we can remove this line completely; per [PyPI](https://pypi.org/project/tensorboard/2.11.0/), this version of tensorboard supports Python >= 3.7 and we don't package anything lower than 3.7 in nixpkgs.
2. Can remove `pythonAtLeast` from the top of the file too.

Reply:
I removed `pythonAtLeast` but I believe the Nixpkgs maintainers generally prefer the `pythonOlder` bound to be present?

Reply:
I don't have a strong opinion and don't mind keeping it there. Thanks for addressing!